lib/server/server.js

```diff
@@ -13,6 +13,7 @@ Push.addListener('token', function(currentToken, value) {
 
 Meteor.methods({
   'raix:push-update': function(options) {
+    this.unblock();
```

Comment:
Why do we need this?

Reply:
@shivang007 good eye. Its been removed. 
On my branch in production, this method can be very slow sometimes. So I didn't want it slowing down other parts of my app.

Reply:
dont worry its been removed from the pull request.

Reply:
@shivang007 if you dont see anything else please approve. if not let me know what you see.